weld/tokenizer.rs

```diff
@@ -92,9 +92,10 @@ pub fn tokenize(input: &str) -> WeldResult<Vec<Token>> {
 
         // Regular expressions for various types of tokens.
         static ref KEYWORD_RE: Regex = Regex::new(
-            "if|for|zip|len|lookup|keyexists|slice|exp|iter|merge|result|let|true|false|macro|\
-             i8|i32|i64|f32|f64|bool|vec|appender|merger|vecmerger|dictmerger|groupmerger|tovec").unwrap();
-
+            "if|for|zip|len|lookup|keyexists|slice|exp|cudf|iter|merge|result|let|true|false|macro|\
```

Comment:
Had to add in cudf to resolve conflicts and be mergeable with master.

I was going to rebase on master, but turns out there are a few breaking changes from the last commit.